test/e2e/serial/tests/scheduler_removal.go

```diff
@@ -71,10 +72,20 @@ var _ = Describe("[serial][disruptive][scheduler][schedrst] numaresources schedu
 
 			dp := createDeploymentSync(fxt, "testdp", serialconfig.Config.SchedulerName)
 
+			dpNName := nroSchedObj.Status.Deployment
+			if dpNName.Namespace == "" || dpNName.Name == "" {
+				Fail(fmt.Sprintf("scheduler deployment missing: %q", dpNName.String()))
+			}
+
 			By(fmt.Sprintf("deleting the NRO Scheduler object: %s", serialconfig.Config.NROSchedObj.Name))
 			err = fxt.Client.Delete(context.TODO(), serialconfig.Config.NROSchedObj)
 			Expect(err).ToNot(HaveOccurred())
 
+			// make sure scheduler deployment is gone
+			config := textlogger.NewConfig(textlogger.Verbosity(1))
```

Comment:
we can and should reuse a single logger for all the use in this group of test, just move it in `Describe`, initialize in the top level `BeforeEach` and use when needed